src/tests/jsonLogicFixtures.js

```diff
@@ -179,204 +198,330 @@ export const schemaWithValidationThatDoesNotExistOnProperty = {
   },
 };
 
-export const multiRuleSchema = {
+export const schemaWithComputedAttributeThatDoesntExist = {
   properties: {
     field_a: {
       type: 'number',
-      'x-jsf-logic-validations': ['a_bigger_than_b', 'is_even_number'],
-    },
-    field_b: {
-      type: 'number',
+      'x-jsf-logic-computedAttrs': {
+        default: 'iDontExist',
+      },
     },
   },
-  required: ['field_a', 'field_b'],
-  'x-jsf-logic': {
-    validations: {
-      a_bigger_than_b: {
-        errorMessage: 'A must be bigger than B',
-        rule: {
-          '>': [{ var: 'field_a' }, { var: 'field_b' }],
+};
+
+export const schemaWithInlinedRuleOnComputedAttributeThatReferencesUnknownVar = {
+  properties: {
+    field_a: {
+      type: 'number',
+      'x-jsf-logic-computedAttrs': {
+        title: {
+          rule: {
+            '+': [{ var: 'IdontExist' }],
+          },
         },
       },
-      is_even_number: {
-        errorMessage: 'A must be even',
-        rule: {
-          '===': [{ '%': [{ var: 'field_a' }, 2] }, 0],
-        },
+    },
+  },
+};
+
+export const schemaWithComputedAttributeThatDoesntExistTitle = {
+  properties: {
+    field_a: {
+      type: 'number',
+      'x-jsf-logic-computedAttrs': {
+        title: `this doesn't exist {{iDontExist}}`,
       },
     },
   },
 };
 
-export const schemaWithMissingValueInlineRule = {
+export const schemaWithComputedAttributeThatDoesntExistDescription = {
   properties: {
     field_a: {
       type: 'number',
       'x-jsf-logic-computedAttrs': {
-        title: {
-          ruleOne: {
-            '+': [1, 2],
-          },
-          ruleTwo: {
-            '+': [3, 4],
-          },
-        },
+        description: `this doesn't exist {{iDontExist}}`,
       },
     },
   },
-  required: [],
 };
 
-export const schemaWithTwoRules = {
+export const schemaWithGreaterThanChecksForThreeFields = {
   properties: {
     field_a: {
       type: 'number',
-      'x-jsf-logic-validations': ['a_bigger_than_b'],
     },
     field_b: {
       type: 'number',
-      'x-jsf-logic-validations': ['is_even_number'],
+    },
+    field_c: {
+      type: 'number',
     },
   },
   required: ['field_a', 'field_b'],
   'x-jsf-logic': {
     validations: {
-      a_bigger_than_b: {
-        errorMessage: 'A must be bigger than B',
+      require_c: {
         rule: {
-          '>': [{ var: 'field_a' }, { var: 'field_b' }],
+          and: [
+            { '!==': [{ var: 'field_b' }, null] },
+            { '!==': [{ var: 'field_a' }, null] },
+            { '>': [{ var: 'field_a' }, { var: 'field_b' }] },
+          ],
         },
       },
-      is_even_number: {
-        errorMessage: 'B must be even',
-        rule: {
-          '===': [{ '%': [{ var: 'field_b' }, 2] }, 0],
+    },
+    allOf: [
+      {
+        if: {
+          validations: {
+            require_c: {
```

Comment:
todo: for readability, i think this should be called `a_bigger_than_b` or not? we need guidelines on how to name these validations 😅  because technically this is not making c required. That's only on the `then` as side effect.

Reply:
changed, good point. Some internal docs i guess around working with JSON schema 🤔

Reply:
I hope you are taking note of these little things for later when we open the PR about json-logic docs ^.^